src/qt/forms/receivecoinsdialog.ui

```diff
@@ -297,6 +297,17 @@
                  </property>
                 </spacer>
                </item>
+               <item>
+                <widget class="QPushButton" name="receivingAddressesButton">
+                 <property name="text">
+                  <string>Receiving Addresses</string>
+                 </property>
+                 <property name="icon">
+                  <iconset resource="../pivx.qrc">
+                   <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
+                 </property>
+                </widget>
```

Comment:
add a third property above the closing `</widget>` line to maintain button icon consistency for the row:

```diff
Index: src/qt/forms/receivecoinsdialog.ui
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/qt/forms/receivecoinsdialog.ui	(date 1535500417000)
+++ src/qt/forms/receivecoinsdialog.ui	(date 1535507981000)
@@ -306,6 +306,9 @@
                   <iconset resource="../pivx.qrc">
                    <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
                  </property>
+                 <property name="autoDefault">
+                  <bool>false</bool>
+                 </property>
                 </widget>
                </item>
               </layout>

```